dvc/command/experiments/run.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import logging
+
+from dvc.command import completion
+from dvc.command.repro import CmdRepro
+from dvc.command.repro import add_arguments as add_repro_arguments
+from dvc.command.repro import append_doc_link
```

Comment:
```suggestion
from dvc.command.base import append_doc_link
```

We should be explicit about where we are importing this method from, otherwise it looks like we are trying to import some `repro` specific implementation of `append_doc_link`.